students/ReemAlqaysi/lesson06/test_mailroom.py

```diff
@@ -1,54 +1,54 @@
 #!/usr/bin/env python3
-
-import mailroom
+import pytest
+from mailroom import *
 import os
 
+# check if we are getting the right text with the right donor and donation amounts
+def test_thank_you_text():
+    fullname = 'Reem Alqaysi'
+    amount = 222
+    letter = "\n\nDear Reem Alqaysi:\n Thank you for your donation of $222, we appriciate your support to our service. \n MailRoom Team\n"
+    try:
+        assert thank_you_text(fullname,amount) == letter
+    except AssertionError:
```

Comment:
you don't typically want to catch the assertion errors and let pytest pick those and report as test failure